packages/react-components/react-search/.babelrc.json

```diff
@@ -0,0 +1,4 @@
+{
+  "extends": "..\\..\\..\\.babelrc-v9.json",
```

Comment:
oh weird, this should be `"extends": "../../../.babelrc-v9.json",`

Reply:
thanks!